ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.19__arm64-nightly.yaml

```diff
@@ -781,6 +781,25 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-disasterrecovery-sanity
     workflow: cucushift-installer-rehearse-gcp-ipi
+- as: baremetal-sno-ipv4-disk-etcd-encryption-f14
+  capabilities:
+  - intranet
+  cron: 38 18 4,11,18,25 * *
+  steps:
+    cluster_profile: equinix-ocp-metal-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:arm64-latest
+    env:
+      AUX_HOST: openshift-qe-metal-ci.arm.eng.rdu2.redhat.com
```

Comment:
Will you be adding `ADDITIONAL_WORKERS: 0` here to skip `ref: upi-install-heterogeneous` step ?

Reply:
ADDITIONAL_WORKERS: 0 is set on WF level, no need